__tests__/version.test.ts

```diff
@@ -298,7 +298,7 @@ describe('When providing multiple different archs', () => {
     expect(macosAsset).toEqual({
       name: 'cmake-3.19.3-Linux-aarch64.tar.gz',
       platform: 'linux',
-      arch: '',
+      arch: 'arm64',
```

Comment:
This `macos` switch is indicative of the "macos == non-x86" issue.  I don't really know how to work around it